Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: An easy way to install armory's agent for kubernetes
 
 type: application
 
-version: 0.1.2
+version: 0.2.1
```

Comment:
What was the reason for bumping from `0.1.2` to `0.2.1`?

Reply:
The breaking config changes? but there wasn't a breaking change in the chart itself so... chalk it up to confusing the two versions 😬

Reply:
Having a chart version and an agent version can be confusing, this is ok for now but we'll try to use the same version of the application as the chart version, provided that helm restrictions allow us to use helm chart versions like `0.7.0-rc.1`